src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -274,7 +275,7 @@ internal void StartMessageBatchReceiver(CancellationToken cancellationToken)
                 {
                     try
                     {
-                        if (!_started || cancellationToken.IsCancellationRequested)
+                        if (cancellationToken.IsCancellationRequested)
                         {
                             _logger.LogInformation("Message processing has been stopped or cancelled");

```

Comment:
The log message still says "has been stopped"

Reply:
Just checked the track 2 code and unfortunately it has the same log message that is not modified to reflect new logic so for consistency we can consider leaving this the same.

Reply:
It is still correct that it was stopped or cancelled, as the cancellation token is signalled when stopping.